fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordContext.java

```diff
@@ -304,6 +304,34 @@ public long getCommittedVersion() {
         return database.asyncToSync(timer, event, async);
     }
 
+    /**
+     * Join a future, following the same logic that asyncToSync uses to validate that the operation isn't blocking
```

Comment:
"asyncToSync" should be a link

I think that comma is extraneous

Reply:
Done.